File: years/2022/clojure/aoc-2022/test/aoc_2022/day_08_test.clj
(ns aoc-2022.day-08-test
  (:require [aoc-2022.day-08 :refer :all]
            [clojure.test :refer :all]))

(deftest day-08-test
  (testing "sample data"
    (is (= 21 (part1 "../../data/08-sample.txt"))))
    ;; (is (= 24933642 (part2 "../../data/08-sample.txt"))))
  (testing "puzzle data"
    (is (= 1859 (part1 "../../data/08.txt")))))
  ;;   (is (= 272298 (part2 "../../data/08.txt")))))

File: years/2025/elixir/test/aoc/day_08_test.exs
defmodule AoC.Day08.Test do
  @moduledoc false

  use ExUnit.Case, async: false

  alias AoC.Day08

  # comment this out to always log to the console
  @moduletag :capture_log

  describe "part 1" do
    test "with sample data" do
      assert Day08.part_1(["../data/08.txt"]) == 1
    end
  end

  describe "part 2" do
    test "with sample data" do
      assert Day08.part_2(["../data/08.txt"]) == 1
    end
  end
end
